stable/stolon/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: stolon
-version: 1.1.2
+version: 1.1.3
```

Comment:
Please bump the minor version to `1.2.0`

Reply:
Fixed